quartz/plugins/transformers/links.ts

```diff
@@ -107,7 +107,7 @@ export const CrawlLinks: QuartzTransformerPlugin<Partial<Options> | undefined> =
 
                   // url.resolve is considered legacy
                   // WHATWG equivalent https://nodejs.dev/en/api/v18/url/#urlresolvefrom-to
-                  const url = new URL(dest, `https://base.com/${curSlug}`)
```

Comment:
let's use the `joinSegments` helper for this

Reply:
I don't know if that's what you meant but looking it up led me to path.join which is, indeed, a lot cleaner than raw concatenation

Reply:
There is a function in `util/path.ts` called `joinSegments`.